sdl-core/src/main/scala/io/smartdatalake/workflow/action/FileOneToOneActionImpl.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Smart Data Lake - Build your data lake the smart way.
+ *
+ * Copyright © 2019-2020 ELCA Informatique SA (<https://www.elca.ch>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.smartdatalake.workflow.action
+
+import io.smartdatalake.config.ConfigurationException
+import io.smartdatalake.definitions.{Environment, SDLSaveMode}
+import io.smartdatalake.util.hdfs.PartitionValues
+import io.smartdatalake.util.misc.PerformanceUtils
+import io.smartdatalake.workflow._
+import io.smartdatalake.workflow.dataobject._
+import org.apache.spark.sql.SparkSession
+
+abstract class FileOneToOneActionImpl extends ActionSubFeedsImpl[FileSubFeed] {
```

Comment:
There's a few classes without a doc. Can you please add a short description ?

Reply:
ok, done